TestAfkakClientIntegration.test_roundtrip_large_request assertion error

Continuing on from #31...

```
FAIL: afkak.test.test_client_integration.TestAfkakClientIntegration.test_roundtrip_large_request
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ciena/afkak/afkak/test/testutil.py", line 88, in wrapper
    return func(self)
  File "/home/travis/build/ciena/afkak/.tox/py27-int-snappy-murmur/lib/python2.7/site-packages/nose/twistedtools.py", line 143, in errback
    failure.raiseException()
  File "/home/travis/build/ciena/afkak/.tox/py27-int-snappy-murmur/lib/python2.7/site-packages/twisted/internet/defer.py", line 1418, in _inlineCallbacks
    result = g.send(result)
  File "/home/travis/build/ciena/afkak/afkak/test/test_client_integration.py", line 128, in test_roundtrip_large_request
    self.assertEqual(len(messages), 1)
AssertionError: 0 != 1
```

Seen once. Python 2.7, Kafka 1.1.1.

--- a/afkak/test/int/test_client_integration.py
+++ b/afkak/test/int/test_client_integration.py
@@ -39,7 +39,7 @@ class TestAfkakClientIntegration(IntegrationMixin, unittest.TestCase):
         message_max_bytes=12 * 1048576,  # 12 MB
     )
     client_kw = dict(
-        timeout=2500,
+        timeout=30000,  # Large timeout because Travis is slow.
     )
 
     @kafka_versions("all")
@@ -112,7 +112,7 @@ class TestAfkakClientIntegration(IntegrationMixin, unittest.TestCase):
         log.debug('Sending FetchRequest')
         # Fetch request with max size of 1 MiB
         fetch = FetchRequest(self.topic, 0, 0, 1024 ** 2)
-        fetch_resp, = yield self.client.send_fetch_request([fetch], max_wait_time=10000)
+        [fetch_resp] = yield self.client.send_fetch_request([fetch], max_wait_time=10000)
         log.debug('Got FetchResponse %r', fetch_resp)
         self.assertEqual(fetch_resp.error, 0)
         self.assertEqual(fetch_resp.topic, self.topic)

--- a/afkak/test/int/test_client_integration.py
+++ b/afkak/test/int/test_client_integration.py
@@ -112,7 +112,7 @@ class TestAfkakClientIntegration(IntegrationMixin, unittest.TestCase):
         log.debug('Sending FetchRequest')
         # Fetch request with max size of 1 MiB
         fetch = FetchRequest(self.topic, 0, 0, 1024 ** 2)
-        fetch_resp, = yield self.client.send_fetch_request([fetch], max_wait_time=1000)
+        fetch_resp, = yield self.client.send_fetch_request([fetch], max_wait_time=10000)
         log.debug('Got FetchResponse %r', fetch_resp)
         self.assertEqual(fetch_resp.error, 0)
         self.assertEqual(fetch_resp.topic, self.topic)

--- a/afkak/test/int/test_client_integration.py
+++ b/afkak/test/int/test_client_integration.py
@@ -109,9 +109,11 @@ class TestAfkakClientIntegration(IntegrationMixin, unittest.TestCase):
         self.assertEqual(produce_resp.partition, 0)
         self.assertEqual(produce_resp.offset, 0)
 
+        log.debug('Sending FetchRequest')
         # Fetch request with max size of 1 MiB
         fetch = FetchRequest(self.topic, 0, 0, 1024 ** 2)
         fetch_resp, = yield self.client.send_fetch_request([fetch], max_wait_time=1000)
+        log.debug('Got FetchResponse %r', fetch_resp)
         self.assertEqual(fetch_resp.error, 0)
         self.assertEqual(fetch_resp.topic, self.topic)
         self.assertEqual(fetch_resp.partition, 0)

